### PR TITLE
Set up tailwind

### LINKED ISSUE #2: Set up : Tailwind Fonts and Colors
## Summary

Extend the Tailwind configuration file with the colors of your liking. Ideally, team should come together to decide on the color schema of the website. Here is a recommendation :
Primary : #0A2239
Secondary : #53A2BE
Black : #0D2225

For the fonts, we can use [Inter](https://rsms.me/inter/) which is also recommended by Tailwind.

### Screenshots
N/A

### Basic example

N/A

### Motivation

Setting up our primary and accent colors in tailwind makes our lives easier by a huge margin since if we want to change a color, we can change it in one place and move on with our lives. Also, deciding on the primary and secondary colors before we start to code and style our project helps all the teams working on the project to have consistent styles.

### PR DESCRIPTION
**Pull request by ayubabdullah**
installing tailwind CSS as a dev dependency, adding three custom colors ( primary, secondary, accent), and adding the inter-font family as a custom font

### CODE REVIEW

**Review by Iseluin — APPROVED** [review]
LGTM, good work 👍🏻

**Review by OussamaMasri — APPROVED** [review]
Great Job Ayub

File: tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,16 @@
...
+        primary: '#0A2239',
+        secondary: '#53A2BE',
+        accent: '#0D2225',
+      },
+      fontFamily: {
+        inter: ['Inter', 'sans-serif'],
+      },
+    },
+    plugins: [],
   },
```
Comment (karzan7899) [inline tailwind.config.js:16]: keep it up and I will hire you for my company. Great work ma boi!

Comment (karzan7899) [inline tailwind.config.js:16]: good Job, and fast enough. keep it up

**Review by karzan7899 — APPROVED** [review]

**Comment by MARWAHAKMI** [comment]
@ayubabdullah, could you please remove the package-lock.json file, because there is no need for this other than everything sounds good. Great job, Ayub.

**Review by MARWAHAKMI — APPROVED** [review]
@ayubabdullah, could you please remove the package-lock.json file, because there is no need for this other than everything sounds good. Great job, Ayub.

**Review by MARWAHAKMI — APPROVED** [review]

**Comment by Iseluin** [comment]
> @ayubabdullah, could you please remove the package-lock.json file, because there is no need for this other than everything sounds good. Great job, Ayub.

It kind of makes sense to add package-lock.json to the .gitignore file, but let me think about that for a while.

**Comment by MARWAHAKMI** [comment]
> > @ayubabdullah, could you please remove the package-lock.json file, because there is no need for this other than everything sounds good. Great job, Ayub.
> 
> It kind of makes sense to add package-lock.json to the .gitignore file, but let me think about that for a while.

The official documentation says that it's recommended to commit lock file. see this: [npm docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-locks)
but I meant that since we're using yarn for this project, it's better to avoid mixing package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. And to avoid these warnings ` warning package-lock.json found. Your project contains lock files generated by tools
other than Yarn. It is advised not to mix package managers in order to avoid resolution 
inconsistencies caused by unsynchronized lock files. To clear this warning, remove
package-lock.json.`

**Review by Iseluin — CHANGES_REQUESTED** [review]
To not to create a conflict between package managers, please use yarn every time you add a dependency, so as Marwa pointed out, please delete the package-lock.json and update your pull request @ayubabdullah , otherwise, great work :1

**Comment by Iseluin** [comment]
> > > @ayubabdullah, could you please remove the package-lock.json file, because there is no need for this other than everything sounds good. Great job, Ayub.
> > 
> > 
> > It kind of makes sense to add package-lock.json to the .gitignore file, but let me think about that for a while.
> 
> The official documentation says that it's recommended to commit lock file. see this: [npm docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-locks) but I meant that since we're using yarn for this project, it's better to avoid mixing package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. And to avoid these warnings ` warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.`

Yup, what I meant was I got to think about including the lock files in general in the commits or not. Because they will create a lot of merge conflicts in the future. Thanks for pointing out the conflicting package managers, I have updated my review.

**Comment by ayubabdullah** [comment]
> 

Yes sir, i will delete it and push it again, because i used to npm that's why in the first i used npm to install tailwind but after that i installed yarn and try to used to it ,

Thanks Marwa, kutay

**Review by Iseluin — APPROVED** [review]

**Pull request merged**